programs/address-lookup-table/src/processor.rs

```diff
@@ -173,13 +173,6 @@ impl Processor {
         let transaction_context = &invoke_context.transaction_context;
         let instruction_context = transaction_context.get_current_instruction_context()?;
 
-        let lookup_table_account =
```

Comment:
you probably need a feature flag for this change

Reply:
Thanks for your feedback @buffalu. As far as I can tell, and according to the tests, existing behaviour doesn't appear to be affected whatsoever. It's just using one less borrow to accomplish the exact same result. Is there something that sticks out to you that indicates that wouldn't be the case? Cheers 🙏

Reply:
actually might not need one looking at the code closer.

anything that creates a potential divergence in behavior in the cluster needs a feature flag

Reply:
Yep, thought so. I definitely don't see any evidence of how this would affect consensus. Just a minor optimisation. Thanks for reviewing.

Reply:
This change could result in updated nodes returning a different instruction error than non updated nodes. That wouldn't break consensus because instruction errors aren't currently included in the bank hash AFAIK. But still good to be on the safe side with a feature flag as @buffalu mentioned.